pkgs/by-name/ca/cassette-framework/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cairo,
+  fontconfig,
+  libxkbcommon,
+  libxcb,
+  xcbutilwm,
+  xcbutilkeysyms,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cassette-framework";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "fraawlen";
+    repo = "cassette";
+    tag = finalAttrs.version;
+    hash = "sha256-+SC/K2vHVXcGesc2AbJS+9rTxEZOoOwBnQZeLUITsjc=";
+  };
+
+  strictDeps = true;
+
+  buildInputs = [
+    libxkbcommon
+    cairo
+    fontconfig
+    libxcb
+    xcbutilwm
+    xcbutilkeysyms
+  ];
+
```

Comment:
add passthru.updateScript?

Reply:
Just added but I never did it before.
Is it well done?